### ISSUE TITLE
question about the future of the phat_small testing example

### ISSUE THREAD

**Issue by lea-hagen** [BEAST-Fitting/beast#661]
For the BEAST testing, we've mostly (but not entirely) moved to using files from metal_small rather than phat_small.  Currently both of them have the same sets of files in `test_regresscheck`, so the cached file names are assigned based on the `cls.dset` setting and then downloaded afterwards.  My question has to do with adding new cached files for new tests (as I'm doing for `compare_spec_type` in https://github.com/BEAST-Fitting/beast-examples/pull/16) - if I add them for metal_small, should I also generate them for phat_small?  Since it seems like we're phasing out phat_small, I'd assume the answer would be no.  However, if that's the case, we'd need to add a bunch of checks that `cls.dset == "metal"` before some downloads and within new test functions.  What would be the best way to proceed?

**Comment by karllark** [comment]
I don't think you need to add them for phat_small.  metal_small is definitely the future and so it would be good to fully phase out phat_small.
